region/lowernorfair/east.json

```diff
@@ -1326,7 +1358,7 @@
                       {"heatFrames": 850},
                       {"ammo": [{
                         "type": "PowerBomb",
-                        "count": 3
```

Comment:
The third PB is to get boosted out of the Morph passage. It can be negated, but that requires either SpringBall or an X-Ray standup +ceiling clip (and I think X-Ray standup isn't a tech we have yet)

Reply:
Not sure why I was changing it for but it was obviously wrong.

Reply:
Reducing the base count, with the extra PB as an alternate to SpringBall, is a good idea though.